tests/helpers/utils.py

```diff
@@ -0,0 +1,28 @@
+import multiprocessing
+import runpy
+import sys
+from os import path as osp
+
+
+def run_main(*args):
+    # patch sys.args
+    sys.argv = list(args)
+    print("Args", args)
```

Comment:
This should be removed. It could interfere with tests

Reply:
Oh yeah, thank you for pointing that out.